draft-ietf-mops-streaming-opcons.md

```diff
@@ -219,6 +288,46 @@ Presentations:
  * MOPS 2020-10-30 Interim meeting:\\
    https://www.youtube.com/watch?v=vDZKspv4LXw&t=17m15s
 
+#Evolution of Transport Protocols and Transport Protocol Behaviors
+
+Because networking resources are shared between users, a good place to start our discussion is how contention between users, and mechanisms to resolve that contention in ways that are "fair" between users, impact streaming media users. These topics are closely tied to the usage of transport protocols and transport protocol behaviors. 
+
+For most of the history of the Internet, the dominant transport protocols in use have been UDP and TCP, and they have each had relatively consistent behaviors, although those behaviors have changed over time. 
+
+## UDP and UDP Protocol Behaviors {#udp-behavior}
```

Comment:
nit: there's a bit of redundancy here and elsewhere in the PR with e.g. "UDP Protocol"

Reply:
Yeah, I was trying to say something that didn't matter. This will be "UDP and Its behavior", and elsewhere.

Reply:
Wfm